types/react-bootstrap/index.d.ts

```diff
@@ -112,6 +112,28 @@ declare namespace ReactBootstrap {
     var ButtonGroup: React.ClassicComponentClass<ButtonGroupProps>;
 
 
+    // <ToggleButtonGroup />
+    interface ToggleButtonGroupProps extends React.HTMLProps<ToggleButtonGroup> {
+        /** Required if `type` is set to 'radio' */
+        name?: string;
+        type: 'radio' | 'checkbox';
+        defaultValue?: any;
+        value?: any;
```

Comment:
Unfortunately, I can't really do that, there is a conflict with React.HTMLProps: https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/260611743#L2006-L2011

I've reverted it back to `any`, but I've also made it clear that `any[]` is valid for `ToggleButtonGroupProps`.